### DISCUSSION TITLE
Support for database views

### DISCUSSION THREAD

**Discussion by kgodey** [mathesar-foundation/mathesar#2281] [Feature Ideas]
Mathesar's concept of Explorations is very similar to database views.

We should support showing database views in the Mathesar UI and integrate them with our Explorations. 

---
*If you'd like us to build this feature, please comment below with more details about how you plan to use this functionality.*

**Comment by AlxGlx** [comment]
Being able to see and interact with database views would be useful. :)
Also, being able to see the count of them in Schemas view, e.g. X Tables, Y Views

**Comment by weetabix** [comment]
I use views a lot for covering regular queries from clients, or to supply statistics. Being able to let my clients see a view of averages, for instance, as a "table" right in mathesar would free up a fair bit of time a week. Even if it's a read-only interface for created views!

**Reply by zackkrida** [reply]
This is useful feedback, thanks @weetabix!

**Comment by acxz** [comment]
With 0.8.0: https://mathesar.org/blog/2025/12/05/release-0-8-0, views have been implemented!
Thanks so much for this feature!
